src/sentry/web/frontend/accept_project_transfer.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import absolute_import
+
+from django import forms
+from django.contrib import messages
+from django.utils.translation import ugettext_lazy as _
+from sentry import roles
+from sentry.web.frontend.base import BaseView
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect, Http404
+from django.utils.encoding import force_str
+from django.core.signing import BadSignature, SignatureExpired
+from sentry.utils.signing import unsign
+from sentry.models import AuditLogEntryEvent, OrganizationMember, Organization, Team, Project
+
+
+class AcceptProjectTransferForm(forms.Form):
+    team = forms.ChoiceField(choices=[])
+
+    def __init__(self, request, *args, **kwargs):
+        super(AcceptProjectTransferForm, self).__init__(*args, **kwargs)
+        teams = []
+        for o in OrganizationMember.objects.filter(user__email=request.user):
```

Comment:
How is this query working? It should be `.filter(user=request.user)`

Also, this is all relatively expensive since it does a bunch of queries to resolve. If someone were a member of a handful of organizations, and if each org had a handful of projects, we're at `O(N*M)` for number of queries.

I think you should refactor this to use `Organization.objects.get_for_user` and `Team.objects.get_for_user`.

Reply:
Oh, I mispoke. You're not querying for projects. So my fault for the confusion.

The value in using the `get_for_user` stuff is because it also applies scoping permissions.

But there's also an extra query being done in here that can be eliminated since `Organization.objects.get_for_user` would handle the `OrganizationMember` bit for you.

Reply:
And for `Team.objects.get_for_user`, you can pass `with_projects=False` to not get Projects.

Reply:
So does this look better?
```python
teams = []
for o in Organization.objects.get_for_user(request.user):
    for t in Team.objects.get_for_user(o, request.user, with_projects=False):
        option = " %s - %s" % (t.name, o.name)
        teams.append([t.id, option])
```

Reply:
Yeah, as long as that works through manual testing, that seems better.